src/js/index.js

```diff
@@ -0,0 +1,102 @@
+// main actions  with keyboard will be there
+import createKeyboardView from './keyboardView';
+import getLocalLanguage from './getLocalLanguage';
+import keyboardPressedEvents from './keyboardPressedEvents';
+import changeToPressed from './changeToPressed';
+import changeToUnpressed from './changeToUnpressed';
+import keyboardUnpressedEvents from './keyboardUnpressedEvents';
+import writeSymbol from './writeSymbol';
+import mouseEvents from './mouseEvents';
+import moveCursor from './moveCursor';
+
+
+createKeyboardView();
+
+const changeRegister = () => {
+
+    document.querySelectorAll('.keyboard__button').forEach((button) => {
+        button.childNodes.forEach((element) => {
+            element.childNodes.forEach((item) => item.classList.toggle('on'))
+        });
+
+    });
+};
+
+
+const changeRegisterByCapsLock = () => {
+
+    document.querySelectorAll('.keyboard__button').forEach((button) => {
+        if (!button.matches('.Digit0') && !button.matches('.Digit1') &&
+            !button.matches('.Digit2') && !button.matches('.Digit3') &&
+            !button.matches('.Digit4') && !button.matches('.Digit5') &&
+            !button.matches('.Digit6') && !button.matches('.Digit7') &&
+            !button.matches('.Digit8') && !button.matches('.Digit9') &&
+            !button.matches('.Digit-') && !button.classList.contains('Equal=') &&
+            !button.matches('.Backslash') && !button.matches('.Slash')) {
+            button.childNodes.forEach((element) => {
+                element.childNodes.forEach((item) => item.classList.toggle('on'))
+            });
+        }
+    });
```

Comment:
BigO() ≈ n * n * n
Could you try to optimize that?

Reply:
Nothing came to mind for optimization :(